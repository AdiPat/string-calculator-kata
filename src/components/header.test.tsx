import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";

describe("Header Tests", () => {
  it("renders a header with a title", async () => {
    const header = await import("./header");
    const { Header } = header;
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
