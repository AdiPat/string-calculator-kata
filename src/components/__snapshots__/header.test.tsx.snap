// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Tests renders a header with a title 1`] = `
<div>
  <h1
    class="text-2xl text-white text-center"
  >
    TDD Kata: String Calculator ðŸŒž
  </h1>
  <p
    class="text-md text-white text-center mt-4"
  >
    This is a simple string calculator that was built using TDD (Test Driven Development).
     
  </p>
</div>
`;
